region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -99,6 +99,82 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Pause Abuse Spark Interrupt (Gain Blue Suit) - Full Platform",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          {"and": [
+            "Morph",
+            "canDodgeWhileShooting",
+            "canCameraManip",
+            "canTrickyJump"
+          ]},
+          "ScrewAttack"
+        ]},
+        "h_RModeCanRefillReserves",
+        {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+        {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
+        {"or": [
+          {"and": [
+            "canBePatient",
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]},
+          {"partialRefill": {"type": "ReserveEnergy", "limit": 5}}
+        ]},
+        {"canShineCharge": {"usedTiles": 29, "openEnd": 0}},
+        {"autoReserveTrigger": {}},
+        "canRModePauseAbuseSparkInterrupt"
+      ],
+      "clearsObstacles": ["A", "B"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Kill the Sidehoppers using only morph and beams or Screw Attack. Using morph and dodging it can be done damage-free.",
+        "Shinecharge on the runway, then take a ceiling thorn hit and pause abuse to spark interrupt."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Pause Abuse Spark Interrupt (Gain Blue Suit) - Samus Eater Shinecharge",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
```

Comment:
I think you should remove the other branch and make this strat be about Crystal Flashing.  Without the CF, its the same as the above strat but harder.